monitor_helper.rb

```diff
@@ -34,7 +34,7 @@ def get_sandbox_space_name(email)
 	def user_space_exists(user_space_name, org_spaces)
 
 		org_spaces.each do |org_space|
-			if org_space["entity"]["name"] == user_space_name
+			if org_space["entity"]["name"].downcase == user_space_name
```

Comment:
Should we also do `user_space_name`, just to account for any spaces that may have been created with mixed-case?